weave

```diff
@@ -1667,7 +1667,31 @@ launch_router() {
            --label=weavevolumes $WEAVEDB_IMAGE >/dev/null
     fi
 
-    RESOLV_CONF=$(chroot ${HOST_ROOT:-/} readlink -f /etc/resolv.conf)
+    echo "Container PATH is $PATH"
+    # Avoid readlink if not present in FHS directories of host
+    if test -x ${HOST_ROOT:-/}/bin/readlink \
```

Comment:
`command_exists` can only check the current environment, not the chroot. I don't know of any standard command that would do it for another root. Something like `chroot ... sh -c "command -v ..."` might work in some cases, even NixOS, but that's executing a binary ("sh") from the host and I'd rather not depend on the host like that.